.travis.yml

```diff
@@ -2,11 +2,19 @@ language: go
 go:
   - 1.8.3
   - 1.9.2
+branches:
+  only:
+    - master
 install: make install-ci
-env:
- # Set higher timeouts for Travis
- - TEST_TIMEOUT_SCALE=20 PACKAGE=github.com/m3db/m3metrics
 sudo: required
 dist: trusty
-script:
- - make all
+env:
+  global:
+    - TEST_TIMEOUT_SCALE=20
+    - PACKAGE=github.com/m3db/m3db
```

Comment:
should this be `github.com/m3db/m3metrics`?

Reply:
Good catch, thanks!